frontend/src/js/amazon.jsx

```diff
@@ -5,70 +5,44 @@ import PaymentStatus from './PaymentStatus/index.jsx'
 import SearchResults from './SearchResults/index.jsx'
 import ProductPage from './ProductPage/ProductPage.jsx'
 import Basket from './BasketEntry/index.jsx'
+import { browserHistory, Router, Route, IndexRoute } from 'react-router'
 import Checkout from './Checkout/index.jsx'
```

Comment:
Oh my bad, I was looking on the train and had a thought that it was possibly on one of the lines above heh